README.md

```diff
@@ -139,7 +141,7 @@ oc cluster up --skip-registry-check --enable=router,sample-templates
 In addition to the deployment, grant the KubeVirt components some additional roles:
 
 ```bash
-oc adm policy add-scc-to-user privileged -n kube-system -z kubevirt-privileged
-oc adm policy add-scc-to-user privileged -n kube-system -z kubevirt-controller
-oc adm policy add-scc-to-user privileged -n kube-system -z kubevirt-apiserver
+oc adm policy add-scc-to-user privileged -n kubevirt -z kubevirt-privileged
```

Comment:
`kubevirt-privileged`, `kubevirt-controller` and  `kubevirt-apiserver` are added to the `privileged` scc by the operator. But we need to do this for `kubevirt-operator` now.

Reply:
+1!